analyze_bak.py

```
@@ -0,0 +1,229 @@
+            chunk = f.read(max_chunk_size)
+            if not chunk:
+                break
+                
+            # Look for table names
+            tables = table_pattern.findall(chunk)
+            for table in tables:
+                try:
+                    table_name = table.decode('utf-8', errors='ignore')
+                    if table_name not in results["tables"]:
+                        results["tables"].append(table_name)
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)